src/getting-started/sw-requirements.md

```diff
@@ -24,7 +24,7 @@ sudo apt install build-essential cmake -y
 
 ### Go Programming Language
 
-Erigon utilizes Go (also known as Golang) version 1.23 or newer for part of its development. It is recommended to have a fresh Go installation. If you have an older version, consider deleting the /usr/local/go folder (you may need to use sudo) and re-extract the new version in its place.
+Erigon utilizes Go (also known as Golang) version 1.24 or newer for part of its development. It is recommended to have a fresh Go installation. If you have an older version, consider deleting the /usr/local/go folder (you may need to use sudo) and re-extract the new version in its place.
```

Comment:
```suggestion
Erigon requires [Go](https://go.dev/) 1.24+ for development.
```

Reply:
I would offload this stuff to the Go website, merge this with the next paragraph on where to go for next steps.